### DISCUSSION TITLE
グループ作成時の問題点と改善希望

### DISCUSSION THREAD

**Discussion by misafortune** [growilabs/growi#7446] [Ideas / Requests] [answered]
現在グループ作成を進めており、かなり苦戦しております。

1. LDAPのグループ連携が使えないので、すべて管理者が手作業でグループツリーを作るしかない（100ユーザーで20グループほど必要です）
2. 管理者が所属していないグループも作成する必要がありますが、その場合ページ作成時に特定のグループの候補に出て来ないので、すべてのグループに管理者を手動で追加する必要があります。

という状況なので、管理者がページを作成する場合、所属しているグループのみが選択出来るのではなく
ページ作成の特定グループ選択時に存在するグループがすべてリストに出てきて選択出来たらかなり楽になると思うので、是非実装して欲しいです。

**Comment by yuki-takei** [comment]
@misafortune 
LDAP からのグループ同期、現在設計中です。
wsgrowi.slack.com の dev-grant チャンネルで開発進捗を確認できます。

**Comment by yuki-takei** [comment]
[/資料/外部仕様/LDAPグループ同期](https://dev.growi.org/646d5637019e02272a42cf86)

**Comment by yuki-takei** [comment, marked as answer]
v6.3.2 で LDAP/Keycloak からのグループ同期が実装されました。
https://docs.growi.org/ja/admin-guide/upgrading/63x.html
